package io.github.rezmike.photon.data.network.error

class NotFoundError() : Exception("Данные не найдены")